src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
+
+As we approach The Merge of Ethereum Mainnet, **you should be on high alert for scams trying to take advantage of users during this transition.** Do not send your ETH anywhere in an attempt to "upgrade to ETH2." There is no "ETH2" token, and there is nothing more you need to do for your funds to remain safe.
+
+[More on Ethereum security](/security/#eth2-token-scam)
+
+### Node operators and dapp developers {#node-operators-dapp-developers}
+
+<ExpandableCard
+title="Staking node operators and providers"
+contentPreview="If you are a staker running your own node setup or a node infrastructure provider, there are a few things you need to be aware of to be prepared for The Merge."
+id="staking-node-operators">
+Key action items include:
+
+- Running _both_ a consensus layer client as well as an execution layer client. Third-party endpoints to obtain execution data will be unavailable after The Merge.
+- Authenticating both execution layer and consensus layer clients with a shared JWT secret so they can securely communicate with one another.
+
+Not completing the above items will result in your node being seen as "offline" after The Merge until both layers are synced and authenticated.
+
+- Setting a `fee recipient` address to direct your earned transaction fee tips/MEV to.
+
+Not setting a `fee recipient` will still allow your validator to behave as usual, but you will miss out on unburnt fee tips and any MEV you would have otherwise earned in blocks your validator proposes.
+
+For more detailed information and a summary of links to client resources, stakers are encouraged to check out the [Merge Readiness Checklist](https://launchpad.ethereum.org/en/merge-readiness/) over on the Staking Launchpad to make sure you're fully prepared for The Merge.
+
+Note for stakers using [SaaS](/staking/saas/) or [staking pools](/staking/pools/): There is nothing you need to do to prepare for The Merge. [More below on staying safe.](#users-holders)
+</ExpandableCard>
+
+<ExpandableCard
+title="Non-validating node operators and infrastructure providers"
+contentPreview="If you're operating a non-validating Ethereum node, the most significant change that comes with The Merge is the requirement to run clients for BOTH the execution layer AND the consensus layer."
+id="node-operators">
+You probably are already running an execution layer client, such as Geth, Erigon, Besu or Nethermind. Up until The Merge, an execution layer client was enough to receive, properly validate, and propagate blocks being gossiped by the network. _After The Merge_, the validity of transactions contained within an execution payload will also depend on the validity of the "consensus block" it is contained within.
```

Comment:
tense switching 

```suggestion
You probably are already running an execution layer client, such as Geth, Erigon, Besu or Nethermind. Until The Merge, an execution layer client will receive, properly validate, and propagate blocks being gossiped by the network. _After The Merge_, the validity of transactions contained within an execution payload will also depend on the validity of the "consensus block" containing the transaction.
```

Reply:
not sure about this now. The validity of the transactions in the execution payload is independent of the beacon block wrapping them. The validity of the transactions is still determined by the execution client re-executing them locally and verifying that no rules are violated. Post-merge, transaction gossip, transaction execution, state-management, EVM etc are all still handled by the execution client.

"You are probably already running an execution client, such as Geth, Erigon, Besu or Nethermind. Until The Merge, the execution client alone is sufficient for interacting with the Ethereum blockchain. After The Merge, the execution client hands over responsibility for block building, block gossip and all consensus logic to a consensus client."